app/javascript/gobierto_data/webapp/components/landingviz/VisualizationsGrid.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div
+    v-if="publicVisualizations.length"
+    class="gobierto-data-visualization--grid"
+  >
+    <template v-for="{ items, config, name, id, columns, slug, datasetName } in publicVisualizations">
+      <router-link
+        :key="id"
+        :to="`/datos/${slug}/v/${id}`"
+        class="gobierto-data-visualizations-name"
+      >
+        <div class="gobierto-data-visualization--card">
+          <div class="gobierto-data-visualization--aspect-ratio-16-9">
+            <div class="gobierto-data-visualization--content">
```

Comment:
When you need this kind of nested HTML structures, it usually recommendable to create an _layout_ component with slots for the content.